src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,91 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+
+type Props = {
+  total: number;
+  perPage: string;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange = () => {},
+}) => {
+  const totalPages = Math.ceil(total / +perPage);
+  const numberOfPages = [];
+
+  for (let i = 1; i <= totalPages; i += 1) {
+    numberOfPages.push(i);
+  }
+
+  const isFirstPage = (): boolean => {
+    if (currentPage === 1) {
+      return true;
+    }
+
+    return false;
+  };
+
+  const isLastPage = (): boolean => {
+    if (currentPage === totalPages) {
+      return true;
+    }
+
+    return false;
+  };
```

Comment:
The function 'isLastPage' can be simplified to return the condition directly without the 'if' statement.
```suggestion
  const isLastPage = currentPage === totalPages;
```